main.js

```diff
@@ -0,0 +1,88 @@
+$('.search').on('click', function () {
+  var cityName = $('#search-query').val();
+
+  $('#search-query').val('');
+
+  fetch(cityName);
+});
+
+var fetch = function (city) {
+ $.ajax({
+    method: 'GET',
+    url: `https://api.openweathermap.org/data/2.5/weather?q=${city}&units=imperial&appid=d4ca3f29ba27c26dac6f9144d941ec22`,
+    dataType: 'json',
+    success: function (data) {
+      renderWeather(data);
+    },
+    error: function (jqXHR, textStatus, errorThrown) {
+      console.log(textStatus);
+    }
+  })
+  $.ajax({
+    method: 'GET',
+    url: `https://api.openweathermap.org/data/2.5/forecast?q=${city}&units=imperial&appid=d4ca3f29ba27c26dac6f9144d941ec22`,
+    dataType: 'json',
+    success: function (data) {
+      renderForecast(data);
+    },
+    error: function (jqXHR, textStatus, errorThrown) {
+      console.log(textStatus);
+    }
+  })
+};
+
+var renderWeather = function (data) {
+  $('.weather').empty();
+
+  
+    const weather = {
+      city: data.name,
+      temp: data.main.temp,
+      condition: data.weather[0].description,
+      icon: data.weather[0].icon
+   };
+    
+    var source = $('#weather-template').html();
+    var template = Handlebars.compile(source);
+    var newHTML = template(weather);
+
+    $('.weather').append(newHTML);
+  };
+
+  
+
+  var renderForecast = function (data) {
+    $('.forecast').empty();
+  
+    //var m = moment('data.list[7].dt', 'dddd');
```

Comment:
This didn't work because you didn't install moment - you'd need to link it in your `index.html` file